core-models/src/main/scala/com/mesosphere/usi/core/models/ResourceRequirement.scala

```diff
@@ -0,0 +1,9 @@
+package com.mesosphere.usi.core.models
+
+import org.apache.mesos.v1.{Protos => Mesos}
+
+trait ResourceRequirement {
```

Comment:
Could you add docs since this is part of our protocol?

Reply:
✅